apps/backend/src/data/migrations/1654699651443-create-store-table.ts

```diff
@@ -0,0 +1,37 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export class CreateStoreTable1654699651443 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'Store',
```

Comment:
keep the table names lower case `store`, `modifier-category`. and the name in kebab case